warehouse/packaging/models.py

```diff
@@ -92,6 +93,10 @@ class Project(db.ModelBase):
         nullable=False,
         server_default=sql.func.now(),
     )
+    updated = Column(
+        DateTime(timezone=False),
+        onupdate=datetime.datetime.now
```

Comment:
What is the benefit? Sounds to me that using ORM here is simpler and doesn't introduce much performance penalty.